### ISSUE TITLE
uniprot-fasta loading problem

### ISSUE THREAD

**Issue by antcsmith** [intermine/intermine#1232]
I can't seem to load sequences for uniprot protein isoforms (for example A0AUZ9-2) HumanMine seems to have the same problem. I'm using the uniprot-fasta source. In my project.xml:
```
<source name="uniprot-fasta" type="fasta">
  		<property name="fasta.taxonId" value="9606 10090 10116 7955 559292 284812"/>
  		<property name="fasta.className" value="org.intermine.model.bio.Protein"/>
  		<property name="fasta.classAttribute" value="primaryAccession"/>
  		<property name="fasta.dataSetTitle" value="UniProt data set"/>
  		<property name="fasta.dataSourceName" value="UniProt"/>
  		<property name="src.data.dir" location="/zeta/data/datasets/mitominer/mitominer4/uniprot/2015_09"/>
  		<property name="fasta.includes" value="uniprot_sprot_varsplic.fasta"/>
  		<property name="fasta.sequenceType" value="protein" />
  		<property name="fasta.loaderClassName" value="org.intermine.bio.dataconversion.UniProtFastaLoaderTask"/>
      </source>
```
I can see the fasta entry in the datafile itself. However, the fasta entries themselves don't have a taxon, just the species name. How does the loader know which to entries to load - is this the problem?

Thanks,

Anthony

**Comment by antcsmith** [comment]
Not sure why it won't let me add the xml but here is screen shot of my project.xml instead

![project_xml](https://cloud.githubusercontent.com/assets/2598621/11249826/248ddcee-8e21-11e5-8c82-a0c80f6e30d2.png)

I should also say I'm using InterMine 1.5.4

**Comment by julie-sullivan** [comment]
@antcsmith Yes you are right! Works on FlyMine (loading the sequence of human isoforms) and works in humanmine if I load that source alone. I will fix!

To go from the name to the taxon Id, we call this method:

https://github.com/intermine/intermine/blob/dev/bio/sources/fasta/main/src/org/intermine/bio/dataconversion/UniProtFastaLoaderTask.java#L48

Which does a look up. So we would expect this to work.

**Comment by julie-sullivan** [comment]
@antcsmith Here's the fix: 2de3dba97a0643c02dc9b5c5c0c35bf9b6872715

It was taking UniProt as the priority data source for sequence, and UniProt said the sequence was NULL. 

Remove that line from the priority file and it should work fine. Thanks for finding this!!

**Comment by antcsmith** [comment]
Thanks!
